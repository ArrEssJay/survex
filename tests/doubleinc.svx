; pos=no warn=3 error=0
*include cmd_dummy
*include cmd_dummy.svx
