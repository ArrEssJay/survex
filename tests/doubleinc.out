In file included from ./doubleinc.svx:2:
./cmd_dummy.svx:1: warning: Reentering an existing survey is deprecated
./cmd_dummy.svx:1: warning: Originally entered here
./cmd_dummy.svx:10: warning: Station already fixed at the same coordinates

Removing trailing traverses...

Concatenating traverses...

Simplifying network...

Calculating network...

Calculating traverses...

Calculating trailing traverses...

Calculating statistics...

Survey contains 1 survey station, joined by 0 legs.
There are 0 loops.
Total length of survey legs =    0.00m (   0.00m adjusted)
Total plan length of survey legs =    0.00m
Total vertical length of survey legs =    0.00m
   1 0-node.
Done.
There were 3 warning(s).
